packages/xrpl/package.json

```diff
@@ -30,6 +30,7 @@
     "ripple-address-codec": "^4.3.0",
     "ripple-binary-codec": "^1.7.0",
     "ripple-keypairs": "^1.3.0",
+    "run-s": "0.0.0",
```

Comment:
Should this be a dev dependency instead of a normal depenency?

Reply:
yes

Reply:
Fixed